cmd/containerd-shim-runhcs-v1/options/runhcs.pb.go

```diff
@@ -26,7 +27,7 @@ var _ = time.Kitchen
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.GoGoProtoPackageIsVersion2 // please upgrade the proto package
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
```

Comment:
Note that this is aligned with containerd.

https://github.com/containerd/containerd/blob/8b9498909dba6dda44fda26d95ef7dbcd1765995/api/events/container.pb.go#L28